### PR TITLE
Update Animal Genomics site navigation

### PR DESCRIPTION
**Pull request by bjlang**

### CODE REVIEW

File: sites/main-site/src/content/special-interest-groups/animal-genomics/index.md [hunk 1]
```diff
@@ -29,4 +29,8 @@ Finally, the group will be a place where users can share their experience with n
 
 # Activities
 
-The format will involve a monthly meeting featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.
+The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.
+
+# How to join
+
+Join the [nf-core Slack](/join#slack) and find us in the [#animal-genomics channel](https://nfcore.slack.com/archives/CE5EL6326)
```
Comment (JoseEspinosa) [inline sites/main-site/src/content/special-interest-groups/animal-genomics/index.md:36]: The link does not take me to the #animal-genomics channel thus, I will delete it
```suggestion
Join the [nf-core Slack](/join#slack) and find us in the #animal-genomics channel
```

File: sites/main-site/src/content/special-interest-groups/animal-genomics/index.md [hunk 2]
```diff
@@ -29,4 +29,8 @@ Finally, the group will be a place where users can share their experience with n
 
 # Activities
 
-The format will involve a monthly meeting featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.
+The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.
```
Comment (JoseEspinosa) [inline sites/main-site/src/content/special-interest-groups/animal-genomics/index.md:32]:
```suggestion
The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will feature a diverse range of topics in the field of animal genomics, including:
* Inspiring talks from top genomics researchers.
* Novel ongoing efforts in animal genomics (new consortiums, new calls, etc).
* More technical sessions on practical examples of how to use nf-core pipelines applied to animal genomics research. 
The discussion will often be a follow-up on the talk.
```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Nice! 😄  Just expanded a little bit the focus of the presentations and small suggestion about the channel link

**Comment by JoseEspinosa** [comment]
@nf-core-bot fix linting

File: sites/main-site/src/content/special-interest-groups/animal-genomics/index.md [hunk 3]
```diff
@@ -29,4 +29,13 @@ Finally, the group will be a place where users can share their experience with n
...
+The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will feature a diverse range of topics in the field of animal genomics, including:
+
+- Inspiring talks from top genomics researchers.
+- Novel ongoing efforts in animal genomics (new consortiums, new calls, etc).
+- More technical sessions on practical examples of how to use nf-core pipelines applied to animal genomics research.
+  The discussion will often be a follow-up on the talk.
+
+# How to join
+
+Join the [nf-core Slack](/join#slack) and find us in the #animal-genomics channel
```
Comment (mashehu) [inline sites/main-site/src/content/special-interest-groups/animal-genomics/index.md:41]:
```suggestion
Join the [nf-core Slack](/join#slack) and find us in the [#animal-genomics channel](https://nfcore.slack.com/archives/C06M8VDEHLG)
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Thanks!

File: sites/main-site/src/content/special-interest-groups/animal-genomics/meetings.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ groupName: Animal Genomics
 
 # Meeting schedule
 
-The Animal Genomics special interest group meets on the third Wednesday
+The [Animal Genomics special interest group](index.md) meets on the third Wednesday
```
Comment (mashehu) [inline sites/main-site/src/content/special-interest-groups/animal-genomics/meetings.md:9]:
```suggestion
The [Animal Genomics special interest group](/special-interest-groups/animal-genomics) meets on the third Wednesday
```

**Review by mashehu — CHANGES_REQUESTED** [review]

**Pull request merged**